src/DevSoc.Eventathon/ClientApp/src/api/health-endpoint.js

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 import useSWR from 'swr';
 
-export function useHealthCheck () {
+export const checkHealth = () => {
```

Comment:
hooks should generally start with `use` so you know it's a hook, e.g. `useHealthCheck`

Reply:
Updated